models/novelsGenres.js

```diff
@@ -1,11 +1,9 @@
 const NovelsGenres = (connection, Sequelize, Genres, Novels) => {
   return connection.define('novelsGenres', {
-    genreId: { type: Sequelize.INTEGER, primaryKey: true, references: { model: Genres, key: 'id' } },
-    novelId: { type: Sequelize.INTEGER, primaryKey: true, references: { model: Novels, key: 'id' } }
+    genreId: { type: Sequelize.INTEGER, references: { model: Genres, key: 'id' } },
+    novelId: { type: Sequelize.INTEGER, references: { model: Novels, key: 'id' } }
```

Comment:
These columns are the primary key of the table, that setting should have stayed.